examples/evaluation/dataset-generator/generate.py

```diff
@@ -0,0 +1,63 @@
+from distilabel.llms import OpenAILLM
```

Comment:
i guess that distilabel is a new dependency. if i'm right requirements need update